includes/wc-payment-api/class-wc-payments-http.php

```diff
@@ -167,7 +167,7 @@ public function start_connection( $redirect ) {
 		$this->connection_manager->enable_plugin();
 
 		// Register the site to wp.com.
-		if ( ! $this->connection_manager->is_registered() ) {
+		if ( ! $this->connection_manager->is_connected() ) {
```

Comment:
Ref: https://github.com/Automattic/jetpack-connection/blob/b6cb5e87ffe1dca5260307066f720f8ddcfebda3/src/class-manager.php#L541

Otherwise, I get this error: 
```
PHP Deprecated:  Automattic\Jetpack\Connection\Manager::is_registered is <strong>deprecated</strong> since version 1.12.1
```